R/module-helpers.R

```diff
@@ -301,7 +301,7 @@ add_carbon_info <- function( data, carbon_info_table, matchvars = c("region", "G
   }
 
   data %>%
-    left_join_error_no_match(carbon_info_table, by = matchvars) %>%
+    left_join(carbon_info_table, by = matchvars) %>%
```

Comment:
What's going on with this change?

Reply:
For land input 1 and 2, everything in the carbon info table has a match.  In land input 3, there start to be some mismatches and NAs. So I had to relax the join here and overwrite the NA's on line 191 (also had to be done in the old DS). I think it was like 3 regions that just don't have one specific land type; Pakistan and I don't remember the other two. I think they were maybe South America